provisioner/ansible/provisioner.go

```diff
@@ -89,6 +89,8 @@ type Config struct {
 	AnsibleEnvVars []string `mapstructure:"ansible_env_vars"`
 	// The playbook to be run by Ansible.
 	PlaybookFile string `mapstructure:"playbook_file" required:"true"`
+	// Specifies --ssh-extra-args on command line defaults to -o IdentitiesOnly=yes
```

Comment:
This code comment will get turned into the documentation automagically if you call `make generate` from inside the project root.